backend/app/controllers/spree/admin/taxons_controller.rb

```diff
@@ -83,6 +84,11 @@ def destroy
       def taxon_params
         params.require(:taxon).permit(permitted_taxon_attributes)
       end
+
+      def resource_not_found
+        flash[:error] = flash_message_for(Taxon.new, :not_found)
+        redirect_to admin_taxonomies_path
+      end
```

Comment:
Seems like we could generalize this logic instead of duplicating it across controllers?